db/migrate/20160907095103_add_availability_to_listing_shape.rb

```diff
@@ -0,0 +1,5 @@
+class AddAvailabilityToListingShape < ActiveRecord::Migration
+  def change
+    add_column :listing_shapes, :availability, :string, limit: 32, default: "none", after: :shipping_enabled
```

Comment:
I read quickly about ActiveRecord Enum. It works so that it emulates the native enum type by converting the allowed values to integers. That's quite ok I guess, but then we lose some clarity (i.e. not easy to remember by just looking at the database value what does 3 or 4 stand for).

I don't thing the native MySQL enum type is supported by Rails.